src/wizard/wizard.html

```diff
@@ -29,25 +29,34 @@ <h3 class="blank-slate-pf-main-action">{{$ctrl.loadingWizardTitle}}</h3>
     <div class="wizard-pf-position-override" ng-transclude ></div>
   </div>
   <div class="modal-footer wizard-pf-footer wizard-pf-position-override" ng-class="{'wizard-pf-footer-inline': $ctrl.embedInPage}">
-    <pf-wiz-cancel class="btn btn-default btn-cancel wizard-pf-cancel"
-                   ng-class="{'wizard-pf-cancel-no-back': $ctrl.hideBackButton}"
-                   ng-disabled="$ctrl.wizardDone" ng-click="$ctrl.onCancel()"
-                   ng-if="!$ctrl.embedInPage">{{$ctrl.cancelTitle}}</pf-wiz-cancel>
+    <button ng-if="!$ctrl.embedInPage"
```

Comment:
Looks good. My only question is, if there is a wizard with more than 2 panels, what happens when the "next" button is clicked?  Does it reset focus on the first item in the next panel?

Reply:
This does not handle setting the focus when a page is shown. That would be left to the application. Barring the application setting the focus, the next button would still have the focus.